### PR TITLE
Initial MVP 10.1 Commit

### PR DESCRIPTION
**Pull request by navancommits**
## Description
Changes compatible with Sitecore 10.1

## Motivation
Since Sitecore 10.1 is the latest version, good to keep up with the latest

## How Has This Been Tested?
Just followed the usual process of running init.ps1 and up.ps1 to see the site running locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

## Checklist:

- [x] I have read the Contributing guide.
- [x] My code follows the code style of this project.
- [x] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change and it requires an update to the documentation.
- [ ] My change is a documentation change and it requires an update to the navigation.

### CODE REVIEW

**Comment by navancommits** [comment]
https://navansitecorenotes.blogspot.com/2021/04/sitecore-mvp-site-running-on-sitecore.html

**Review by robearlam — CHANGES_REQUESTED** [review]
Thanks for contributing!

Mostly looks good, but I added some comments to the changes you made to init.ps1.

If you have improvements that can be suggested for that process they should be lodged as a separate PR as they are unrelated to the 10.1 upgrade.

**Review by navancommits — COMMENTED** [review]
Hi Rob, I wanted to come back to this pr and make it cleaner but you beat me to it! Thanks for the review... This runs fine after the changes....

File: init.ps1 [hunk 1]
```diff
@@ -76,49 +75,67 @@ try {
...
 Add-HostsEntry $cdHost
 Add-HostsEntry $idHost
 Add-HostsEntry $renderingHost
 
-#########################################################
-# Tell git to ignore changes to .env
-# Note: For future upgrades this can be undone by running
-# git update-index --no-assume-unchanged .env
-#########################################################
-git update-index --assume-unchanged .env
```
[the remark this answers is not in the document]
Comment (navancommits) [reply init.ps1]: Done

File: init.ps1 [hunk 2]
```diff
@@ -76,49 +75,67 @@ try {
...
+Set-DockerComposeEnvFileVariable "SITECORE_ID_CERTIFICATE" -Value (Get-SitecoreCertificateAsBase64String -DnsName "localhost" -Password (ConvertTo-SecureString -String $idCertPassword -Force -AsPlainText))
+
+# SITECORE_ID_CERTIFICATE_PASSWORD
+Set-EnvFileVariable "SITECORE_ID_CERTIFICATE_PASSWORD" -Value $idCertPassword
+
+# SITECORE_LICENSE
+Set-EnvFileVariable "SITECORE_LICENSE" -Value (ConvertTo-CompressedBase64String -Path $LicenseXmlPath)
+
+Set-DockerComposeEnvFileVariable "HOST_LICENSE_FOLDER" -Value $LicenseXmlFolderPath
+
```
[the remark this answers is not in the document]
Comment (navancommits) [reply init.ps1]: Done

File: init.ps1 [hunk 3]
```diff
@@ -17,6 +17,11 @@ Param (
 
 $ErrorActionPreference = "Stop";
 
+$cmHost = "mvp-cm.sc.localhost"
+$cdHost = "mvp-cd.sc.localhost" 
+$idHost = "mvp-id.sc.localhost"
+$renderingHost = "mvp.sc.localhost"
+
```
[the remark this answers is not in the document]
Comment (navancommits) [reply init.ps1]: Done

**Pull request merged**